addons/wowl/static/src/services/user.ts

```diff
@@ -33,8 +33,9 @@ export const userService = {
       uid: info.uid,
       allowed_company_ids: user_companies.allowed_companies.map(([id]) => id),
     };
-    return {
-      context: context,
+
+    return Object.assign({}, localizationParameter, {
```

Comment:
I think it is a bad idea to do it like this. This is not defensive programming. If localizationParameter contains additional keys, they will be there. Usually not a big deal, but this is a low level service, should be as clean/safe as possible.

In my opinion, you should here define explicitely each key:
```ts
return {
  timeFormat: ...,
  multiLang: ...,
  ...  
};
```